schema.sql

```diff
@@ -46,6 +47,7 @@ CREATE TABLE role (
     department_id INT,
     band_id INT,
     summary MEDIUMTEXT,
+    sharepoint_link VARCHAR(500),
```

Comment:
Minor comment: I'd just call this `job_spec_url` or something like that. `sharepoint_link` is very vague. Also, ensure that a URL won't ever be longer than 500 characters.